src/main/scala/geotrellis/qatiles/RoadTags.scala

```diff
@@ -4,6 +4,16 @@ import geotrellis.vectortile.{VString, Value}
 
 /** Container for relevant OSM tags on road features */
 case class RoadTags(highway: Option[String], surface: Option[String]) {
+
+  // TODO: Promote this check and underlying highwayCalcValues to
+  //       application specific logic somewhere up in
+  //       PopulationNearRoads
+  def isIncludedRoad: Boolean = highway match {
+    case Some(v) if RoadTags.highwayIncludedValues.contains(v) => true
+    case Some(_) => false
+    case None => false
+  }
```

Comment:
This is just application of `roadFilter: RoadTags => Boolean`, including this method here is a little confusing given that its not used consistently elsewhere.

Reply:
Oof. Yeah it is. Pushed a commit that cleans this up a bit and addresses your other related comment.